File: FlightManagementSystem.Domain/Flight.cs
namespace FlightManagementSystem.Domain
{
    public class Flight
    {
        public int Id { get; set; }
        public string FlightNumber { get; set; }
        public DateTime DateDeparture {  get; set; }
        public string PlaceDeparture { get; set; }
        public string PlaceArrival { get; set; }
        public AircraftType AircraftType { get; set; }
    }
}

File: FlightManagementSystem.Presentation/Client/flights/src/models/flight.ts
export interface Flight {
    id: number | undefined,
    flightNumber: string,
    dateDeparture: string | undefined,
    placeDeparture: string,
    placeArrival: string,
    aircraftType: AircraftType
  }
  

  export enum AircraftType {
    Embraer,
    Boeing,
    Airbus
  }
